scripts/base/frameworks/cluster/main.zeek

```diff
@@ -270,7 +270,7 @@ export {
 	##
 	## Returns: a topic string that may used to send a message exclusively to
 	##          a given cluster node.
-	global node_topic: function(name: string): string;
+	global node_topic: function(name: string): string &redef;
```

Comment:
The current computation of `node_topic()` and `nodeid_topic()` for Broker is fully in script land using `/` to separate components of topics. That doesn't work well with NATS which uses `.` as subject separator. For now the idea is that loading NATS specific policy scripts can switch the implementation to use `.` instead via the redef.

I gather your proposal is to go via a bif that then does `backend->NodeTopic(...)` or so, I could see that, but having it in script land seemed convenient. Backend specific scripts exist due to various configuration options and some custom events, so that seemed convenient.

If there's stronger opinions coming up, I'm happy to remove it before v8.0 again.

Reply:
I'll drop both commits, they are specific enough to carry separately. Thanks.